app/waf/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build with:
+#   docker build . -t f5devcentral/f5-wafaas-openstack:latest
+
+# upload image with (authorized user only)
+#   docker push f5devcentral/f5-wafaas-openstack:latest
+
+FROM node:11.6.0
+
+LABEL maintainer="F5 Networks Inc. Corp."
+EXPOSE 3000/tcp
+
+WORKDIR /root/app_dir
+ADD dist dist
+RUN rm -r dist/test
+ADD public public
+COPY package.json index.js ./
+RUN npm install -g nc
```

Comment:
Needn't this one, if we needn't to test db connection.

Reply:
although we add `depends_on` in docker-compose.yml, docker-compose has no strategy to make sure to start waf app after postgres is ready. That means compose doesn't know if postgres is ok or not.
As mentioned before, better way is update our waf app to check db connection before start.